tools/build_cdn.js

```diff
@@ -125,7 +125,7 @@ async function installLanguages(languages, options) {
 
   await Promise.all(
     languages.filter((l) => l.third_party)
-      .map((lang) => buildDistributable(lang, options))
+      .map(async(lang) => await buildDistributable(lang, options))
```

Comment:
Out of curiosity, what's the reason for using async/await here?

Reply:
Uncertainty about how async works? :-) Are you going to tell me it's effectively the same either way?